Makefile

```diff
@@ -12,9 +16,9 @@ ifneq ($(RELEASE_VERSION),)
 	VERSIONLDFLAG := -X github.com/tektoncd/cli/pkg/cmd/version.clientVersion=$(RELEASE_VERSION)
 endif
 
-FLAGS := $(VERSIONLDFLAG)$(SKIPLDFLAG)
+FLAGS := $(VERSIONLDFLAG)$(SKIPLDFLAG)$(NAMESPACEFLAG)
 ifneq ($(SKIPLDFLAG),)
-	FLAGS := $(VERSIONLDFLAG) $(SKIPLDFLAG)
+	FLAGS := $(VERSIONLDFLAG) $(SKIPLDFLAG)$(NAMESPACEFLAG)
```

Comment:
I think we need an if case for this also, there may be scenarios of only namespace flag provided

Reply:
I am not sure i understand completely like in which scenario we need if condition 

and in the current code also i see with and without if condition `flags` are same i mean without any condition flags are [here](https://github.com/tektoncd/cli) and with condition flags are [here](https://github.com/tektoncd/cli/blob/master/Makefile#L17) and both are same correct me if i mistaken something